packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -127,9 +344,9 @@ export abstract class GatewayRouteSpec {
 
 class HttpGatewayRouteSpec extends GatewayRouteSpec {
   /**
-   * The criterion for determining a request match for this GatewayRoute.
+   * The criterion for determining a request match for this GatewayRoute
    *
-   * @default - matches on '/'
+   * @default - matches on prefix with '/'
```

Comment:
Just kill these docs (module-private classes don't need documentation on their properties).

Reply:
Removed the doc.